layers/+lang/python/packages.el

```diff
@@ -334,6 +334,9 @@
         "sR" 'spacemacs/python-shell-send-region-switch
         "sr" 'python-shell-send-region)
 
+      ;; we call python-indent-guess-indent-offset ourselves so set to nil to avoid duplicate work
+      (setq-default python-indent-guess-indent-offset nil)
```

Comment:
Can't this be:
```
(setq-default python-indent-guess-indent-offset python-spacemacs-indent-guess)
```
?

Reply:
We could then probably cut out manually calling the function.

Reply:
No, this exactly the bug that is fixed by this pr: since `python-default` changes `tab-width` and runs after `python-mode-hook` (so after `python-mode` has already initialized itself), setting `python-indent-guess-indent-offset` to `t` will be buggy: since it runs with the wrong `tab-width` it calculates a wrong value (it needs to run after we set the tab-width).